neuroquery/tests/test_img_utils.py

```diff
@@ -63,7 +64,13 @@ def test_coordinates_to_maps():
             "z": [27.0, 0.0, 30.0, 17.0, 177.0],
         }
     )
-    maps, masker = img_utils.coordinates_to_maps(coords)
+    if persist_memmap:
+        memmap = tmp_path.joinpath("maps.dat")
+    else:
+        memmap = None
```

Comment:
I'm used to the `memmap = tmp_path.joinpath("maps.dat") if persist_memmap else None` as I find it more readable than the if loop spanned over 4 lines, wdyt?

Reply:
you're right that is better!